sktime/forecasting/compose/_pipeline.py

```diff
@@ -1354,7 +1354,7 @@ def __init__(
         behaviour="update",
         columns=None,
         fit_behaviour="use_actual",
-        forecaster_X_exogeneous="None",
```

Comment:
I would leave this a `str` - regardless of what one might think of this choice, it would have to go through a deprecation cycle if we change it.

Reply:
sure let me leave it as it is